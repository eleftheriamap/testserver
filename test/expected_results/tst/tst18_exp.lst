
test/exp/test/gen/tst/tst18_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	516babde 	sub	w30, w30, #0xaea, lsl #12
   4:	d13f837b 	sub	x27, x27, #0xfe0
   8:	ea5b2fdf 	tst	x30, x27, lsr #11
   c:	8a000000 	and	x0, x0, x0
