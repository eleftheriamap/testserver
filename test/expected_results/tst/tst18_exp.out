Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = fffffffffffff020
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 00000000ff516000
PC     = 000000000000000c
PSTATE : ----
Non-zero memory:
0x00000000 : 0x516babde
0x00000004 : 0xd13f837b
0x00000008 : 0xea5b2fdf
0x0000000c : 0x8a000000
